hw10.py

```diff
@@ -0,0 +1,54 @@
+# 1. До завдання, в якому ви реалізовували телефонну книгу, додати валідацію номера телефону за допомогою RegEx.
+# Врахувати можливість декількох форматів: +380XXXXXXXXX, 380XXXXXXXXX, 0XXXXXXXXX
+import json
+import re
+enter = input('''You can see all contacts(stats), all names(list), someone\'s number(show) or (add), (delete) contacts.
+Enter your command: ''')
+if enter == 'add':
```

Comment:
Here should be the same phonebook as in previous tasks, with a loop. Update this, please